src/USER-MISC/pair_e3b.cpp

```diff
@@ -0,0 +1,688 @@
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   http://lammps.sandia.gov, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+
+   contributing authors: Steven E Strong and Nicholas J Hestand
+   contact: stevene.strong at gmail dot com
+------------------------------------------------------------------------- */
+
+#include "pair_e3b.h"
+
+#include "atom.h"
+#include "neighbor.h"
+#include "neigh_request.h"
+#include "neigh_list.h"
+#include "force.h"
+#include "comm.h"
+#include "memory.h"
+#include "error.h"
+#include "update.h"
+#include "domain.h"
+#include "citeme.h"
+
+#include <cmath>
+#include <cstdio>
+#include <cstdlib>
+#include <cstring>
+
+//these are defined here to avoid confusing hardcoded indicies, but
+//they do not allow flexibility. If they are changed the code will break
+#define DIM 3
+#define NUMH 2  //number of hydrogen atoms per water molecule
+#define NUMO 2  //number of oxygen atoms per pair of water molecules
+#define BOND_DELTA 1.01   //buffer for OH bonds that aren't perfectly rigid
+
+using namespace LAMMPS_NS;
+/* ---------------------------------------------------------------------- */
+
+PairE3B::PairE3B(LAMMPS *lmp) : Pair(lmp),pairPerAtom(10)
+{
+  single_enable = 0;
+  restartinfo = 0;
+  one_coeff = 1;
+  manybody_flag = 1;
```

Comment:
@sstrong99 is there a particular reason, why you set the manybody_flag to 1 here? it is meant to trigger a warning when a manybody potential (where bonded interactions are implicit) is used with explicit bonds. in your examples the explicit bonds, however, are intentional, so the warning may confuse people.

Reply:
No, I only set that because it seemed most consistent. But it looks like the flag isn't really used for anything except that warning. So I agree that it doesn't need to be set. I will fix that.